packages/form-layout/test/form-item.test.js

```diff
@@ -118,4 +118,126 @@ describe('form-item', () => {
       expect(input.getBoundingClientRect().width).to.be.above(testInput.getBoundingClientRect().width + 10);
     });
   });
+
+  describe('required indicator', () => {
+    it('should be hidden from screen readers', () => {
+      expect(item.shadowRoot.querySelector('[part="required-indicator"]').getAttribute('aria-hidden')).to.equal('true');
```

Comment:
nit: I would prefer adding a snapshot test instead. Not a blocker.

Reply:
Added a snapshot test for the form item default state. Removed the default state checks and one level of unnecessary `describe` from the unit tests